mailer.js

```diff
@@ -130,7 +136,8 @@ module.exports = function (log) {
         link: link,
         oneClickLink: oneClickLink,
         supportUrl: this.supportUrl,
-        supportLinkAttributes: this._supportLinkAttributes()
+        supportLinkAttributes: this._supportLinkAttributes(),
+        passwordChangeLinkAttributes: this._initiatePasswordChange()
```

Comment:
added a note to https://github.com/mozilla/fxa-auth-mailer/issues/94#issuecomment-148462202